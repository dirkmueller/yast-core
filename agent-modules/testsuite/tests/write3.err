[ag_modules] ModulesAgent.cc(Write):242 Obsolete interface, don't use any more!
[YCP] tests/write3.ycp:2 alias.al3 true
[ag_modules] ModulesAgent.cc(Write):288 Obsolete interface, don't use any more!
[YCP] tests/write3.ycp:3 alias.al3.comment true
