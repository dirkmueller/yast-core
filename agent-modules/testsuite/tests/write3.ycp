{
    y2warning("alias.al3 %1",SCR::Write(.alias."al3", "A3"));
    y2warning("alias.al3.comment %1",SCR::Write(.alias."al3".comment, "# new alias comment"));

    return true;
}
