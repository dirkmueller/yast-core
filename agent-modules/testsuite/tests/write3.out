(true)
# file with only comments
#

# new alias comment
alias al3 A3
